esx_service/cli/vmdkops_admin.py

```diff
@@ -788,13 +788,13 @@ def generate_vm_list(vms_uuid):
     """ Generate vm names with given list of vm uuid"""
     # vms_uuid is a list of vm_uuid
     # example: vms_uuid=["vm1_uuid", "vm2_uuid"]
-    # the return value is a string like this vm1,vm2
+    # the return value is a string like this vm1,vm2.
     res = ""
     for vm_uuid in vms_uuid:
         vm_name = vmdk_utils.get_vm_name_by_uuid(vm_uuid)
-        res = res + vm_name
-        res = res + ","
-
+        if vm_name:
```

Comment:
I would add a comment- "If VM is already deleted or migrated, vm_name could be NULL and will be skipped".